css/styles.css

```diff
@@ -0,0 +1,115 @@
+header {
+  background-color: #562a81;
+  padding: 2px;
+}
+
+.header h3 {
+  color: white;
+  padding-left: 1rem;
+  font-weight: 100;
+  font-family: "Proza Libre", serif;
+  font-style: normal;
+  letter-spacing: 1.3px;
+}
+
+.user-entry {
+  background-color: #f0f0f0;
+  padding: 1rem;
+}
+
+.user-entry h2 {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 600;
+  margin: 0;
+}
+
+.user-entry img {
+  display: block;
+  padding: 0.5rem 0;
+  width: 100%;
+  border-radius: 1rem;
+}
+
+.user-entry label,
+.user-entry input,
+.user-entry textarea {
+  display: block;
+  width: 100%;
+}
+
+.user-entry label {
+  font-family: "Open Sans", sans-serif;
+  padding-top: 1.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.user-entry input,
+.user-entry textarea {
+  border: 1px solid #c5c6c5;
+  border-radius: 0.2rem;
+  padding: 0.3rem;
+}
+
+.user-entry button {
+  display: block;
+  margin: 1rem 0;
+  background-color: #562a81;
+  color: white;
+  padding: 0.3rem 01rem;
+  border-color: transparent;
+  border-radius: 0.3rem;
+}
+
+.form-actions {
+  display: flex;
+  justify-content: flex-end;
+}
+
+@media only screen and (min-width: 580px) {
+  body {
+    background-color: #f0f0f0;
+  }
+
+  .header {
+    width: 70%;
+    margin: auto;
+  }
+
+  .user-entry {
+    width: 70%;
+    margin: auto;
+  }
```

Comment:
why are you setting `width` directly? you should be using `column-` classes to regulate element's size, and using the `container` class to restrict the size of the page as a whole